kotlin/vector-tiled-layer-custom-style/README.metadata.json

```
@@ -0,0 +1,46 @@
+{
+    "category": "TODO",
+    "description": "Load ArcGIS vector tiled layers using custom styles.",
+    "formal_name": "ArcgisVectorTiledLayerCustomStyle",
+    "ignore": false,
+    "images": [
+        "vector-tiled-layer-custom-2.png",
+        "vector-tiled-layer-style.png",
+        "vector-tiled-layers-list.png"
+    ],
+    "keywords": [
+        "tiles",
+        "vector",
+        "vector basemap",
+        "vector tile package",
+        "vector tiled layer",
+        "vector tiles",
+        "vtpk",
+        "ArcGISVectorTiledLayer",
+        "ExportVectorTilesTask",
+        "ItemResourceCache",
+        "Map",
+        "VectorTileCache"
+    ],
+    "language": "kotlin",
+    "provision_from": [
+        "TODO"
+    ],
+    "provision_to": [
+        "TODO"
+    ],
+    "redirect_from": [
+        "TODO"
+    ],
+    "relevant_apis": [
```

```suggestion
    ],
    "relevant_apis": [
```

redirect not needed on new samples (this is used on the developer website to catch old samples that may have had their name changed)